point_pattern.py

```diff
@@ -119,9 +148,26 @@ def average_nearest_neighbor_distance(points):
      Measure of Spatial Relationships in Populations. Ecology. 35(4)
      p. 445-453.
     """
+
+    #create empty list of distances
+    shortestDistList = []
+
+    for i in points:
+        shortest = 9999999999
+        for j in points:
+            if i!=j:
```

Comment:
Checkout `enumerate`.  This will allow you to catch two coincident points by checking positional index, instead of coordinates.